app/assets/v2/js/grants/new.js

```diff
@@ -5,8 +5,21 @@ $(document).ready(function() {
     web3.eth.net.isListening((error, connectionStatus) => {
       if (connectionStatus)
         init();
+        document.init = true
     });
   }
+  setTimeout(function(){
+      if(!document.init){
+        $('#connect_metamask_error').css('display', 'block');
```

Comment:
for some reason i can't access `show_error_banner()` here... maybe a js variable scoping issue? because its defined in document.ready